index.html

```diff
@@ -326,7 +335,38 @@ <h2>Cards</h2>
       </div>
     </section>
 
-  </div>
+    <section id='images'>
+      <h2>Images</h2>
+      <p>Responsive image and round image.</p>
+      <div class='examples'>
+        <div class="img-container">
+          <img src="https://picsum.photos/400/300" />
+        </div>
+        <div class="round-img-container">
+          <img src="https://picsum.photos/400/300" class="round-img" />
+        </div>
+      </div>
+    </section>
+
+    <section id='input'>
+      <p>There are 4 different types of inputs. </p>
+      <div class='examples'>
+        <div class='input-container'>
+          <input class="input input-error" placeholder='enter email-id'/>
```

Comment:
here what's the type of input? make sure to provide input type in input tag since you have mentioned placeholder as email-id so the input type should be email.

eg: 
<input type="email">

Reply:
changed. thanks.